test/integration/cds_integration_test.cc

```diff
@@ -142,7 +146,7 @@ class CdsIntegrationTest : public Grpc::DeltaSotwIntegrationParamTest, public Ht
 };
 
 INSTANTIATE_TEST_SUITE_P(IpVersionsClientTypeDelta, CdsIntegrationTest,
-                         DELTA_SOTW_GRPC_CLIENT_INTEGRATION_PARAMS);
+                         DELTA_SOTW_GRPC_CLIENT_LAZYINITSTATS_INTEGRATION_PARAMS);
```

Comment:
per below I think maybe it's overkill to add a lazy-init-stats dimensions to *all* the tests,

Reply:
We did talk offline but I am not convinced it's worth the cost. The trigger for me is you increased the sharding.

I feel like this is a signal that we are not relying enough on unit testing; we are depending too much on integration tests.

Reply:
heh. The code if you read is pretty “simple " already: send some request and reading some data, TBH I dont think abstract the common piece outside of the test would help with readability.

Let me remove the extra parameter, together with the "direct " stats integration test pieces. This way the tests will look less messy hopefully.

Reply:
@jmarantz here